src/Apps/Settings/Routes/Shipping/Components/SettingsShippingAddressForm.tsx

```diff
@@ -134,10 +141,39 @@ export const SettingsShippingAddressForm: FC<SettingsShippingAddressFormProps> =
         handleChange,
         handleBlur,
         setFieldValue,
+        setFieldError,
         isValid,
         isSubmitting,
         submitForm,
       }) => {
+        const handlePhoneNumberChange = async (region, number) => {
+          if (region && number && relayEnvironment) {
+            const phoneInformation = await getPhoneNumberInformation(
```

Comment:
Can the component just do this for us?

There's no real reason that consumers of the component should have to perform the query.

The phone number input could also handle the debouncing (if it doesn't already).

Reply:
yes debouncing and moving the call to PhoneNumberInput are my next steps. 

the only thing that bothers me is how to access `isValid` property in this component (that we get through `getPhoneNumberInformation`).

Reply:
Just pass it along in as an argument in some custom `onChange` callback?

Reply:
I feel like we need `isValid` here to tell Formik it's ok.

Reply:
> Just pass it along in as an argument in some custom onChange callback?

will do! thanks for the idea.